terragrunt/prod/terragrunt.hcl

```diff
@@ -35,7 +35,7 @@ generate "inputs" {
   iit_lambda_code_bucket_key_version = "m2WQ7ixHMkV7WZ8HDH0JeJNDFuiOGvzV"
   master_node_instance_count = 3
   master_node_instance_type = "c6g.large.elasticsearch"
-  data_node_instance_count = 12
+  data_node_instance_count = 14
```

Comment:
The data_node_volume_size can be bumped up instead.

Reply:
@mbystedt how about a count of 12 and size of 350